packages/validators/src/raw/core.js

```diff
@@ -42,12 +42,13 @@ export const len = (value) => {
 
 /**
  * Regex based validator template
- * @param {RegExp} expr
+ * @param {Array|RegExp} expr
  * @return {function(*=): boolean}
  */
 export function regex (expr) {
```

Comment:
what if we allow multiple parameters to be provided? Pretty common pattern in native prototype functions.
```suggestion
export function regex (...args) {
```

Reply:
yeah i think it is actually better, i will commit changes then. thanks